### PR TITLE
update README regarding TypeScript (#420)

### PR DESCRIPTION
**Pull request by jbdemonte**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +49,14 @@ sanitize-html is not written in Typescript and there is no plan to directly supp
 
 But, perhaps you'd like to display sanitized HTML immediately in the browser for preview. Or ask the browser to do the sanitization work on every page load. You can if you want to!
 
-* Clone repository and install via npm
-* Run npm install and:
+* Install it using npm  (or yarn):
```
Comment (abea) [inline README.md:52]: Let's say, "Install the package:"

**Review by abea — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -26,9 +26,20 @@ HTML comments are not preserved.
...
 sanitize-html is intended for use with Node.js and supports Node 10+. All of its npm dependencies are pure JavaScript. sanitize-html is built on the excellent `htmlparser2` module.
 
-### Regarding Typescript
+### Regarding TypeScript
 
-sanitize-html is not written in Typescript and there is no plan to directly support it. There is a community supported implementation, [`@types/sanitize-html`](https://www.npmjs.com/package/@types/sanitize-html), however. Any questions or problems while using that implementation should be directed to its maintainers as directed by that project's contribution guidelines.
+sanitize-html is not written in TypeScript and there is no plan to directly support it.  
+There is a community supported typing definition, [`@types/sanitize-html`](https://www.npmjs.com/package/@types/sanitize-html), however.
+```bash 
+yarn add --dev @types/sanitize-html
```
[the remark this answers is not in the document]
Comment (abea) [reply README.md:34]: I don't mind including both below, but here we don't need both and don't prioritize yarn.

**Comment by abea** [comment]
@boutell Any thoughts before we merge?

**Review by abea — COMMENTED** [review]
Holding for @boutell 's review

**Review by boutell — APPROVED** [review]

**Review by abea — APPROVED** [review]

**Pull request merged**